configtx/example_test.go

```diff
@@ -132,21 +132,21 @@ func Example_basic() {
 }
 
 // This example updates an existing orderer configuration.
-func ExampleConfigTx_SetOrdererConfiguration() {
+func ExampleUpdatedOrdererGroup_SetConfiguration() {
 	baseConfig := fetchChannelConfig()
 	c := configtx.New(baseConfig)
 
 	// Must retrieve the current orderer configuration from block and modify
 	// the desired values
-	orderer, err := c.OrdererConfiguration()
+	orderer, err := c.UpdatedConfig().Orderer().Configuration()
 	if err != nil {
 		panic(err)
 	}
 
 	orderer.Kafka.Brokers = []string{"kafka0:9092", "kafka1:9092", "kafka2:9092"}
 	orderer.BatchSize.MaxMessageCount = 500
```

Comment:
This is where embedding is good; makes it easy to change the updated version. It's modification of the original that I worry about but it may not be a problem...